microprofile/7.1/faulttolerance/4.1/25.0.0.7-beta-cl250620250508-1902-MicroProfile-Fault-Tolerance-4.1.1-Java11-TCKResults.adoc

```diff
@@ -0,0 +1,436 @@
+   org.eclipse.microprofile.fault.tolerance.tck.visibility.retry.RetryVisibilityTest.serviceBaseROMRetryMissingOnMethod Passed!
+   org.eclipse.microprofile.fault.tolerance.tck.visibility.retry.RetryVisibilityTest.serviceBaseROM Passed!
+   org.eclipse.microprofile.fault.tolerance.tck.visibility.retry.RetryVisibilityTest.arquillianBeforeClass Failed!
+   org.eclipse.microprofile.fault.tolerance.tck.visibility.retry.RetryVisibilityTest.serviceBaseROMNoRedefinition Passed!
+   org.eclipse.microprofile.fault.tolerance.tck.visibility.retry.RetryVisibilityTest.serviceDerivedClassNoRedefinition Passed!
+   org.eclipse.microprofile.fault.tolerance.tck.visibility.retry.RetryVisibilityTest.serviceBaseROMOverridedClassLevelNoMethodOverride Passed!
+   org.eclipse.microprofile.fault.tolerance.tck.visibility.retry.RetryVisibilityTest.serviceOverrideMethodLevelUsesMethodLevelAnnotation Passed!
+   org.eclipse.microprofile.fault.tolerance.tck.visibility.retry.RetryVisibilityTest.serviceBaseROMOverridedMethodLevel Passed!
+   org.eclipse.microprofile.fault.tolerance.tck.visibility.retry.RetryVisibilityTest.serviceOverrideClassLevelUsesClassLevelAnnotation Passed!
+   org.eclipse.microprofile.fault.tolerance.tck.visibility.retry.RetryVisibilityTest.arquillianAfterClass Failed!
+   org.eclipse.microprofile.fault.tolerance.tck.visibility.retry.RetryVisibilityTest.serviceRetryRemovedAtMethodLevel Passed!
+   org.eclipse.microprofile.fault.tolerance.tck.visibility.retry.RetryVisibilityTest.serviceOverrideClassLevelUsesClassLevelAnnotationWithMethodOverride Passed!
+   org.eclipse.microprofile.fault.tolerance.tck.visibility.retry.RetryVisibilityTest.baseRetryServiceUsesDefaults Passed!
+Test suite: org.eclipse.microprofile.fault.tolerance.tck.metrics.CircuitBreakerMetricTest 2025-05-21T07:56:08 BST
+Tests:3 Failures:0 Errors:2
+   org.eclipse.microprofile.fault.tolerance.tck.metrics.CircuitBreakerMetricTest.arquillianAfterClass Failed!
+   org.eclipse.microprofile.fault.tolerance.tck.metrics.CircuitBreakerMetricTest.arquillianBeforeClass Failed!
+   org.eclipse.microprofile.fault.tolerance.tck.metrics.CircuitBreakerMetricTest.testCircuitBreakerMetric Passed!
+Test suite: org.eclipse.microprofile.fault.tolerance.tck.FallbackExceptionHierarchyTest 2025-05-21T07:56:08 BST
+Tests:29 Failures:0 Errors:2
+   org.eclipse.microprofile.fault.tolerance.tck.FallbackExceptionHierarchyTest.serviceAthrowsE0 Passed!
+   org.eclipse.microprofile.fault.tolerance.tck.FallbackExceptionHierarchyTest.serviceBthrowsE1S Passed!
+   org.eclipse.microprofile.fault.tolerance.tck.FallbackExceptionHierarchyTest.serviceAthrowsException Passed!
+   org.eclipse.microprofile.fault.tolerance.tck.FallbackExceptionHierarchyTest.serviceBthrowsRuntimeException Passed!
+   org.eclipse.microprofile.fault.tolerance.tck.FallbackExceptionHierarchyTest.arquillianBeforeClass Failed!
```

Comment:
Test failures

Reply:
There are lots of failure lines in Fault Tolerance, three in jwt and one in telemetry, but they're all arquillian[Before|After][Class|Suite] methods, suggesting a test setup failure rather than an actual problem. We'll need to check the logs and see what's gone wrong here.